packages/mobile/src/home/WalletHome.tsx

```diff
@@ -147,6 +159,11 @@ export class WalletHome extends React.Component<Props> {
     )
   }
 
+  migrateAccount = () => {
+    this.setState({ isMigrating: true })
```

Comment:
nit: Though there's no bug now, we never bring this component state back to false, which could be confusing in the future. This should probably be handled by the redux store.